[N9wvgpQR] Вынести фото сравнения на клиент

AlexeyFrunza opened the pull request:
-- сжал все фото до нужного размера в формате webp
-- закинул их в папку public/imgs/comparisons
-- поправил компонент сравнения и виджет для работы с локальными фотографиями
-- удалил папку entities/comparison/api

Commit:
[N9wvgpQR] Вынести фото сравнения на клиент
- Добавить фото в public/imgs/comparisons папку на клиенте
-- сжал фотографии для сравнений
-- добавил_такую_перменную

Commit:
[N9wvgpQR] Вынести фото сравнения на клиент
- Добавить фото в public/imgs/comparisons папку на клиенте
-- сжал фотографии для сравнений
-- закинул их в папку public/imgs/comparisons

Commit:
[PMVFvYgp] Вынести фото сравнения на клиент
- Переделать компоненты для сравнения на использование локальных фото
-- вынес все пути к изображениям в массив объектов imgs
-- отфильтровал массив, получив набор слайдов для нужного пути

Commit:
[PMVFvYgp] Вынести фото сравнения на клиент
- Удалить папку entities/comparison/api т.к. теперь отзывы лежат на клиенте
-- удалил папку entities/comparison/api

weekkkk commented on src/entities/comparison/ui/slider/ui/slide/index.vue:
```diff
@@ -11,7 +11,7 @@ const props = defineProps<ComparisonSliderSlideProps>();
   >
     <img
-      :src="`data:image/jpeg;base64,${image}`"
+      :src="`${image}`"
```
`image` и так строка

weekkkk commented on src/widgets/comparison/ui/sliders/index.vue:
```diff
@@ -1,18 +1,75 @@
+      "/restoration/shoes"
+    ],
+    beforeImages: [
+      `${config.app.baseURL}imgs/comparisons/travis-a.webp`,
```
лучше вынести добавление `baseUrl` в компонент, где `<img src=...`

weekkkk commented on src/widgets/comparison/ui/sliders/index.vue:
```diff
@@ -1,18 +1,75 @@
-const beforeImages = computed(() =>
-  data.value?.map(({ beforeImage }) => beforeImage)
-);
+type RouteImages = {
```
вынести в отдельный файл

Commit:
[N9wvgpQR] Вынести фото сравнения на клиент
- Удалить папку entities/comparison/api т.к. теперь отзывы лежат на клиенте
-- вынес {config.app.baseUrl} в компонент , где прописывается src атрибут
-- вынес тип RouteImages в отдельную папку types

Commit:
[N9wvgpQR] Вынести фото сравнения на клиент
- Удалить папку entities/comparison/api т.к. теперь отзывы лежат на клиенте
-- вынес {config.app.baseUrl} в компонент , где прописывается src атрибут
-- вынес тип RouteImages в отдельную папку types
